### ISSUE TITLE
Could you update `graceful-fs` to eliminate `Please update to graceful-fs@^4.0.0 as soon as possible.` warning?

### ISSUE THREAD

**Issue by aszegedi** [allure-framework/allure-jasmine-plugin#9]
The warning message:
```
[91mnpm[0m[91m [0m[91mWARN[0m[91m [0m[91mdeprecated[0m[91m graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```
after `npm ls graceful-fs` I got:
```
└─┬ jasmine-allure-reporter@1.0.0
  └─┬ allure-js-commons@1.1.4
    ├─┬ file-type@2.6.0
    │ └─┬ meow@3.7.0
    │   └─┬ read-pkg-up@1.0.1
    │     └─┬ read-pkg@1.1.0
    │       └─┬ load-json-file@1.1.0
    │         └── graceful-fs@4.1.4 
    └─┬ fs-extra@0.18.4
      └── graceful-fs@3.0.8 
```
Thank you for your help and support!

**Comment by just-boris** [comment]
@aszegedi that was dependency of allure-js-commons. 

I released a new version 1.1.6 with fixed dependency and checked this with mocha-allure.

@ctapobep now it is time to bump dependency here.

**Comment by aszegedi** [comment]
@just-boris Thank you, this was very promt!

**Comment by ctapobep** [comment]
Updated the version and released 1.0.1 of jasmine-allure-reporter.
